databricks/koalas/frame.py

```diff
@@ -232,13 +232,15 @@ def applymap(self, func):
 
         >>> def str_len(x) -> int:
         ...     return len(str(x))
+
         >>> df.applymap(str_len)
            0  1
         0  3  4
         1  5  5
 
         >>> def power(x) -> float:
         ...     return x ** 2
+
```

Comment:
So .. I was thinking we just loose the condition - inserting a newline for two line preparation is optional ..